spyder/plugins/editor/lsp/transport/main.py

```diff
@@ -109,16 +116,21 @@ def restore(self):
 
 if __name__ == '__main__':
     logger_init(args.transport_debug)
+    unknownargs = [x for x in unknownargs if len(x) > 0]
+    unknownargs = ' '.join(unknownargs)
```

Comment:
What does it mean `unknownargs`? Could we use a better variable name instead?

Reply:
Such args are passed to the stdio server, when the transport layer is on charge of spawning it. Could it be ``extra_args``?

Reply:
Yep, much better :+1: